diff -up avidemux2-master/avidemux/qt4/CMakeLists.txt.omv~ avidemux2-master/avidemux/qt4/CMakeLists.txt
--- avidemux2-master/avidemux/qt4/CMakeLists.txt.omv~	2025-11-01 17:39:46.473932731 +0100
+++ avidemux2-master/avidemux/qt4/CMakeLists.txt	2025-11-01 17:39:52.344859786 +0100
@@ -265,7 +265,7 @@ TARGET_LINK_LIBRARIES(avidemux3_${QT_EXT
 #
 # i18n
 #
-ADD_SUBDIRECTORY(i18n)
+#ADD_SUBDIRECTORY(i18n)
 # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # #
 # Install
 # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # #
