--- avidemux_2.6.4/avidemux/common/ADM_render/CMakeLists.txt.omv~	2013-06-17 12:45:47.725688179 +0200
+++ avidemux_2.6.4/avidemux/common/ADM_render/CMakeLists.txt	2013-06-17 12:46:41.715013219 +0200
@@ -20,7 +20,7 @@ ENDIF (APPLE)
 
 add_compiler_export_flags()
 ADD_LIBRARY(${ADM_LIB} SHARED ${${ADM_LIB}_SRCS})
-TARGET_LINK_LIBRARIES( ${ADM_LIB} ADM_core6 ADM_coreUI6 ADM_coreImage6)
+TARGET_LINK_LIBRARIES( ${ADM_LIB} ADM_core6 ADM_coreUI6 ADM_coreImage6 -lX11 -lXext)
 
 IF (USE_OPENGL)
 	ADD_SOURCE_CFLAGS(GUI_render.cpp " -I${QT_HEADERS_DIR}")
--- avidemux_2.6.4/avidemux/gtk/ADM_userInterfaces/ui_support.cpp.omv~	2013-06-17 12:44:21.499766151 +0200
+++ avidemux_2.6.4/avidemux/gtk/ADM_userInterfaces/ui_support.cpp	2013-06-17 12:44:54.491353700 +0200
@@ -8,6 +8,7 @@ extern "C" int getMainNSWindow(void);
 #else
 #include <gdk/gdkx.h>
 #endif
+#include "ADM_toolkitGtk.h"
 
 #include "config.h"
 #include "ADM_inttype.h"
@@ -68,4 +69,4 @@ void getMainWindowHandles(long int *hand
 }
 
 void GUI_OpenApplicationLog() { }
-void GUI_OpenApplicationDataFolder() { }
\ No newline at end of file
+void GUI_OpenApplicationDataFolder() { }
